### DISCUSSION TITLE
Game list hiding parenthesized text, causing issue with region and version selection

### DISCUSSION THREAD

**Discussion by emperordahc** [OnionUI/Onion#1400] [Support] [answered]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.0-RC-b66469de

### Miyoo Firmware Version

202303262339

### Provide a clear and concise description of the issue

The game list is now hiding all text enclosed by parenthesis. This causes issues when trying to play specific region or versions of files that are using standardized file naming for use with scrapers, etc. The games can still be launched.

For instance, the files:

- Super Mario Bros. 3 (Europe).7z
- Super Mario Bros. 3 (Japan) (Rev 1).7z
- Super Mario Bros. 3 (Japan).7z
- Super Mario Bros. 3 (USA) (Rev 1).7z
- Super Mario Bros. 3 (USA).7z

All appear as:

- Super Mario Bros. 3
- Super Mario Bros. 3
- Super Mario Bros. 3
- Super Mario Bros. 3
- Super Mario Bros. 3

Photo: example listing of files missing parenthesized text:

![mi1](https://github.com/OnionUI/Onion/assets/109972325/b86eb664-7b5c-4b1a-a04f-507630b658c7)

Photo: The parenthesized information is also hidden in the detail view.

![mi2](https://github.com/OnionUI/Onion/assets/109972325/d92cb61f-d1a0-4c28-8cef-fa3d11143a41)

### Steps to Reproduce

Refresh the cached files and view the game list.

### Provide any additional context or information that may help us investigate your issue

I had upgraded to the newest beta version of Onion, but had not refreshed files; all region information was visible at that time.

I then refreshed the file list, and all parenthesized information disappeared. The Recents menu section still shows the parenthesized filenames of games launched before the recache, and the menu items work there.

I searched documentation and threads for any settings for this feature, and could not find any.

**Comment by Aemiii91** [comment, marked as answer]
Seems you've run the "Generate miyoogamelist with digest names" tool in Tweaks. Do you have a `miyoogamelist.xml` file in `Roms/FC` ?

**Comment by emperordahc** [comment]
As far as I know, I've never ran that tool. (I just now dug through Tweaks and found the tool; I don't recognize it at all.)

I checked that folder, and, yes, the miyoogamelist.xml file existed. I deleted it, and rescanned the folder; all my files are appearing as normal now. Thanks! I'll dig through all the other folders and delete the XML files there, too. (Is there a way to undo this other than manually deleting the XML files?)

Is there the possibility that the digest names tool is automatically getting triggered during install/another time? I'm sure I've never ran the tool manually, and I often update my files.

**Reply by Aemiii91** [reply]
No, it's purely manual. It will create miyoogamelist.xml files for all systems. We have even thought about adding a warning before you run it.

**Reply by emperordahc** [reply]
I guess I hit it manually somehow! No clue! Appreciate the help! 😀

UPDATE: cleaned the other folders' XML files out, and everything is back to normal. Only about half of the folders had XML files.
